### ISSUE TITLE
Footer: links mix internal/external and https vs http

### ISSUE THREAD

**Issue by svx** [plone/ploneorg.core#161]
The link to the roadmap:
![porg_links-mix](https://cloud.githubusercontent.com/assets/358860/15332780/4aadaa1a-1c67-11e6-93cb-c580e6c30925.png)

The link is http not https and there is external linking used ?

**Comment by polyester** [comment]
yeah the footer had to be written before the DNS switch, so it was all a bit messed up. Now that the new site is live, those can be changed.
